src/domains/notifications/types.ts

```diff
@@ -0,0 +1,4 @@
+export type createNotificationData = {
```

Comment:
I prefer to name types with `PascalCase`
```suggestion
export type CreateNotificationData = {
```